app/assets/stylesheets/common/_markus.scss

```diff
@@ -1366,3 +1363,54 @@ canvas {
   min-width: 250px;
   width: 50%;
 }
+
+.exam-crop-container {
+  display: flex;
+  height: 600px;
+  width: 1000px;
+}
+
+#exam-crop-img-container {
+  background-color: #f1f1f1;
+  flex: 1;
+  margin-right: 5px;
+  overflow-x: auto;
+  overflow-y: auto;
+  text-align: center;
+}
+
+.exam-crop-button-container {
+  display: flex;
+  flex-direction: column;
+  padding-left: 5px;
+  padding-top: 5px;
+  width: 40px;
+}
+
+#increase-crop-scale {
+  margin-bottom: 2px;
+}
+
+.crop-scale-button {
+  height: 40px;
+  margin-top: 2px;
+  min-width: 40px;
+  padding: 0;
+  transform: translate(-70px, 0);
+  width: 40px;
+  z-index: 800; // jcrop uses z-index of 690
+}
+
+.crop-scale-button:hover {
```

Comment:
I know I said to use `:hover`, but also we should make the style consistent with other buttons in MarkUs, including the ones on the page (e.g. the "save" button). So assuming that styling is already applied consistently across all buttons, you should probably remove this rule.